apis/compute/v1alpha1/serverset_types.go

```diff
@@ -94,13 +94,8 @@ type ServerSetTemplateNIC struct {
 	DHCPv6 *bool `json:"dhcpv6"`
 	// +kubebuilder:validation:Optional
 	VNetID string `json:"vnetId,omitempty"`
-	// +kubebuilder:validation:Required
-	Reference string `json:"reference"`
-}
-
-// ServerSetTemplateVolumeMount are the configurable fields of a ServerSetTemplateVolumeMount.
-// It is used to mount a volume to a server.
-type ServerSetTemplateVolumeMount struct {
+	// If created using crossplane, MUST be created before serverset
+	//
```

Comment:
```suggestion
	// The Referenced Volume must be created before the ServerSet is applied
```

But why? Are we not able to lookup this resource?

Reply:
Wanted to use a LanConfig like we use for datacenterconfig, but there's a Crossplane issue that is not yet fixed with mutiple configsin the same CR